### ISSUE TITLE
bundle.min.js:95 Uncaught TypeError: Cannot set property 'dir' of undefined

### ISSUE THREAD

**Issue by YuxinChou** [ipeychev/alloyeditor-react-component#7]
Hi!
When I use it, I encountered some problems:

> Uncaught TypeError: Cannot set property 'dir' of undefined
> Uncaught TypeError: Cannot read property 'add' of undefined
> 
> Uncaught SyntaxError: Unexpected token <
> <!DOCTYPE html>   index.html

I used it 
`import` AlloyEditorComponent from './AlloyEditor' 
`<AlloyEditorComponent` container='editable' />

and index.html

`<script>
    window.ALLOYEDITOR_BASEPATH = 'alloyeditor/';
    window.CKEDITOR_BASEPATH = 'alloyeditor/';
 </script>`
`<link href="alloyeditor/assets/alloy-editor-ocean-min.css" rel="stylesheet">`

**Comment by ipeychev** [comment]
Hello,

Probably the path does not resolve properly to the directory where AlloyEditor is installed. You may take a look on the Network tab of the DevTools of the browser you are using and to check for any unresolved requests. This should tell you more about the reason.

You also may take a look [here](https://github.com/ipeychev/alloyeditor-react-component/issues?q=is%3Aissue+is%3Aclosed) people used to ask similar questions.

Hope that helps!

Thanks,

**Comment by Rohan-Doshi** [comment]
I am facing the similar error. How did you resolve it @YuxinChou

**Comment by Rohan-Doshi** [comment]
I am using hashHistory(react-router) in my app for routing. I have a tab where I need to use alloy-editor.
I referred [this](https://github.com/ipeychev/alloyeditor-react-component) repo to set it up.I am running into exactly similar error. I was able to narrow down the error to routing. When I changed from hashHistory to browserHistroy, it started to work fine(without any change related to alloy-editor part). 
Is alloy-editor not compatible with hashHistory ?
